### DISCUSSION TITLE
[CICLE] Day and Night

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2814] [Ideas]
I think it would be interesting to have day and night cycles in the game.

Initially we can put the sun to follow the server time just for aesthetics, it would be cool. But in the future we can use this variation for new interactions in the game.

Thank you.

**Comment by TheDcol** [comment]
It would be cool I agree :)

Do you think crops could be affected by day/night cycles?

**Reply by future2back** [reply]
Yes, not only crops, but other mechanisms within the game can be affected like, enemies, pests, drops, any event. But I think this should be well studied so as not to negatively affect players from different timezone.

With this idea, we can implement a system with the four seasons of the year in the game, namely, winter, spring, summer, and autumn. These could also impact the in-game harvests.

Hope this gets implemented, expanding the possibilities within the game.


Thank you.
